### DISCUSSION TITLE
Standalone releases

### DISCUSSION THREAD

**Discussion by padvincenzo** [padvincenzo/silence-speedup#9] [Q&A] [answered]
Hi everyone
Thank you for downloading and using this software. I'd like to ask your feedbacks about the (not)standalone version:

* How useful do you think it is to have **both** standalone and non-standalone versions, for each OS?
* Is the difference between the two versions _understandable_?

If not useful, nor understandable, I could just release only the version with FFmpeg included.

I'm looking forward to your feedback

**Comment by rexelbartolome** [comment, marked as answer]
I'm not sure if it's all that useful. I guess it's not the same as each OS having an installed SDK like Java and having the OS a very consistent `Environment Variable` to that like `JAVA_HOME`. I guess I'm more familiar with downloading open source software now that I understand enough what "standalone" means and I already know myself that I've installed FFmpeg before. But I'm sure that if I were myself maybe 5 years ago, I would have never understood what each meant. 

I would say either a documentation part explaining the difference between the two or just releasing the standalone version would make sense. I'm more leaning towards just releasing the standalone version, it's only 30MB difference anyway. And for people that are looking into, recording, storing, and cutting silence from videos, they probably already have a dedicated storage of 500GB - 1TB for their own videos, so a 30MB difference in the program doesn't sound too bad.

**Reply by padvincenzo** [reply]
Thanks @rexelbartolome for your feedback, this is a great point of view. Starting from release 1.2.5, there won't be 2 types of versions, but I'll publish only the `Standalone` ones, though removing the term standalone as it is not useful anymore.
